2.train-VAE/2.fetch_hyper_params.ipynb

```diff
@@ -67,13 +67,13 @@
      "output_type": "stream",
```

Comment:
<p>Please consider adding comments that explain what these loops do</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>